pcweb/whitelist.py

```diff
@@ -1,30 +1,21 @@
-"""A list of whitelist paths that should be built.
-If the list is empty, all pages will be built.
+"""Whitelist for pages to build."""
```

Comment:
revert all changes in pcweb/whitelist.py, this file doesn't need any changes to support temporal docs